src/DssDirectDepositAaveDai.sol

```diff
@@ -381,20 +467,21 @@ contract DssDirectDepositAaveDai {
     function cull() external {
         require(vat.live() == 1, "DssDirectDepositAaveDai/no-cull-during-shutdown");
         require(live == 0, "DssDirectDepositAaveDai/live");
-        require(_add(tic, tau) <= block.timestamp, "DssDirectDepositAaveDai/early-cull");
+        require(_add(tic, tau) <= block.timestamp || wards[msg.sender] == 1, "DssDirectDepositAaveDai/early-cull");
```

Comment:
nit: change message to `"DssDirectDepositAaveDai/unauthorized-cull"`

Reply:
Fixed.